### ISSUE TITLE
Floating prompt after Paladin trigger

### ISSUE THREAD

**Issue by tanderson11** [mtgred/netrunner#4790]
If the runner has a Paladin companion with 3 credits on it at the end of the turn, they must either trash an installed card or trash Paladin. If they trash another installed card, the corp is stuck waiting on a prompt even after the runner has resolved the action.

**Comment by NoahTheDuke** [comment]
Thanks for the report!

**Comment by lostgeek** [comment]
Duplicate of #4784
